Automattic/wp-calypso
# Hide the masterbar during the checkout process from simple to Atomic site.

#### Changes proposed in this Pull Request
The navigation bar (aka masterbar) should not be visible during the loading/checkout process from a simple site being upgraded to an Atomic site. Once the 'Thank You' modal appears the masterbar should reappear.

This PR hides the masterbar from view during the loading process and reappears it at the 'thank you' modal.

#### Testing instructions

* Create a new simple site.
* Purchase an eCommerce plan to upgrade your simple site to an Atomic site.
* During the checkout loading screen, no navigation bar should be visible at the top of the screen.

#### Before (masterbar visible)
These gifs are sped up. Testing in real-time takes ~30seconds.
![masterbar-visible](https://user-images.githubusercontent.com/140841/124188928-90379780-da85-11eb-9803-ea3364479697.gif)

#### After (masterbar hidden)
Droplr link to the after screencast: https://d.pr/i/eqjTmO
